common-build/rules.mk

```diff
@@ -50,7 +50,7 @@ PYTHON          = python3
 PIP             = $(PYTHON) -m pip
 SPHINXBUILD     = $(ENV_DIR)/bin/sphinx-build
 SPHINXAUTOBUILD = $(ENV_DIR)/bin/sphinx-autobuild
-AUTOBUILD_OPTS  = --re-ignore '^(?!.+\.rst$$)' --open-browser --delay 0
+AUTOBUILD_OPTS  = --watch $(TOP_DIR)/src --re-ignore '^(?!.+\.(?:rst|md|mmd|html|css|js)$$)' --open-browser --delay 0
```

Comment:
From a quick look: https://pypi.org/project/sphinx-autobuild/ --re-ignore are regular expression to match files that are ignored from the watch, is this your goal?

Reply:
The regex has an exclamation mark, I think the idea is to ignore everything except files with the extensions listed

Reply:
ok, thx! I've missed it.